src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -1,9 +1,65 @@
 [
+  {
+    "name": "AppServiceApiApp_DisableRemoteDebugging",
+    "description": "Remote debugging should be turned off for API Apps",
+    "recommendation": "Remote debugging should be turned off",
+    "helpUri": "https://github.com/Azure/template-analyzer/blob/main/docs/built-in-bpa-rules.md/#remote-debugging-should-be-turned-off-for-api-apps",
+    "evaluation": {
+      "resourceType": "Microsoft.Web/sites",
+      "where": {
+        "path": "kind",
+        "regex": "api$"
+      },
+      "allOf": [
+        {
+          "resourceType": "Microsoft.Web/sites/config",
+          "where": {
+            "path": "name",
+            "regex": "web$"
+          },
+          "allOf": [
```

Comment:
I'm reading in the documentation that allOf and anyOf are like a list `and` and `or`, why do we use it for lists with only one element?

Reply:
That's currently the only way to create an additional evaluation, especially when `where` is specified.  It's the only way to specify a new resourceType/path/etc.

Reply:
I don't understand, we can't use `where` without anyOf/allOf? Maybe we should add to the documentation information on how to write rules using `where`? It sounds very anti-intuitive to me, that's something that we plan to improve in the future?

Reply:
`where` needs some other operator to complete the evaluation, and usually a structured operator makes the most sense.  The `evaluate` operator is typically what would work best, but hasn't been implemented.  Until that is supported, `allOf`/`anyOf` are effective work-arounds.  The documentation update is in the johnathon/readmeUpdate branch.